src/sss_client/pam_sss.c

```diff
@@ -995,7 +995,7 @@ static int eval_response(pam_handle_t *pamh, size_t buflen, uint8_t *buf,
                     break;
                 }
 
-                if (type == SSS_PAM_CERT_INFO && pi->cert_user == '\0') {
+                if (type == SSS_PAM_CERT_INFO && *pi->cert_user == '\0') {
```

Comment:
may I know why? operator structure access member `.` and operator structure access member through pointer `->` have the highest priority. I do not think that brackets improve readability here. Brackets there are redunnadnd the same as in following example :-)`c = ( (a) + (b) )`

If we want to change that then I would rather prefer `pi->cert_user[0] == '\0'`
but such type of checking is not used anywhere in the module. Therefore I didn't introduce it.